src/overlays/actors/ovl_En_Bigpo/z_en_bigpo.h

```diff
@@ -5,15 +5,66 @@
 
 struct EnBigpo;
 
-typedef void (*EnBigpoActionFunc)(struct EnBigpo* this, GlobalContext* globalCtx);
+typedef void (*EnBigPoActionFunc)(struct EnBigpo*, GlobalContext*);
+
+typedef struct EnBigpoFireParticle {
+    /* 0x00 */ Vec3f pos;
+    /* 0x0C */ LightNode* light;
+    /* 0x10 */ LightInfo info; // size 0xE
+} EnBigpoFireParticle; // size = 0x20
+
+#define ENBIGPO_LIMBCOUNT 10
 
 typedef struct EnBigpo {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0xBC];
-    /* 0x0200 */ EnBigpoActionFunc actionFunc;
-    /* 0x0204 */ char unk_204[0x194];
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ Vec3s limbDrawTbl[ENBIGPO_LIMBCOUNT];
+    /* 0x1C4 */ Vec3s transitionDrawTbl[ENBIGPO_LIMBCOUNT];
+    /* 0x200 */ EnBigPoActionFunc actionFunc;
+    // need to know what func_801A2E54 does to know what this is
+    /* 0x204 */ u8 unkBool204;
+    // idleTimer gets reused:
+    // * when idle flying around, frames until next attack
+    // * when spinning back into reality, counts frames from start
+    // * in death one, counts frames from death start
+    // * after scoop spawned, idle timer used to count down to actor disapear
+    /* 0x206 */ s16 idleTimer; // frame counter
```

Comment:
This kind of documentation is great, but I would suggest to move them out of the struct, we try to avoid comments between structs members.